packages/iov-bns/src/decode.spec.ts

```diff
@@ -394,6 +396,54 @@ describe("Decode", () => {
       const tx = parseTx(decoded, chainId);
       expect(tx.transaction).toEqual(sendTxJson);
     });
+
+    it("decode multisig transaction", () => {
+      const decoded: codecImpl.bnsd.ITx = {
+        signatures: [
+          {
+            sequence: 0,
+            pubkey: {
+              ed25519: fromHex("c9df7bcba2238bedcc681e8b17bb21c1625d21d285b70c20cf53fdd473db9dfb"),
+            },
+            signature: {
+              ed25519: fromHex(
+                "7e899078b35b2d301893b82b0d840f4e1ceb715a86f4d28b2e88312ed605fea83d6b37f7ced4c8023a0d41f79ac9d5556df72c5ec58e26d7db4a6f8c6a537d0a",
+              ),
+            },
+          },
+        ],
+        multisig: [fromHex("8877665544332211")],
```

Comment:
Question: is 8 bytes the real-workd length to expect?

Reply:
Yes, the IDs are sequential and covered by https://github.com/iov-one/weave/blob/22678889d3e1750c21574c340aebf8c7f09a7679/orm/sequence.go#L62-L78

Reply:
Oh, that's interesting. Then we might even want to encode them numerically in https://github.com/iov-one/iov-core/issues/1216 like we do with proposals.

If they are sequential, we can use `number`s in the safe integer range and throw an error for anything else.